test/services/notifications/setup/batch-notifications.service.test.js

```diff
@@ -367,6 +369,30 @@ describe('Notifications Setup - Batch notifications service', () => {
         }
       ])
     })
+
+    it('should update the "event.metadata.error"', async () => {
+      await BatchNotificationsService.go(testRecipients, determinedReturnsPeriod, referenceCode, journey, eventId)
+
+      const updatedResult = await EventModel.query().where('id', eventId).first()
```

Comment:
Use `findById()`. It would be more idiomatic for this context, and returns just a single result automatically.

```suggestion
      const updatedResult = await EventModel.query().findById(eventId)
```